src/main/java/frc/robot/commands/MoveMovableSubsystem.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.MoveableSubsystem;
+
+public class MoveMovableSubsystem extends CommandBase {
+  private MoveableSubsystem subsystem;
+  private BooleanSupplier isFinished;
+  private DoubleSupplier power;
+
+  /**
+   * This command can move any subsystem that implements MovableSubsystem without
+   * an isFinished
```

Comment:
without an isFinished
really?? I am a programmer who didn't write the code. What should I understand from this??